wg/bsi.md

```diff
@@ -0,0 +1,55 @@
+# Cloud Native Batch System Initiative Working Group Charter
+ 
+## Introduction
+
+Cloud Native Batch System Initiative Working Group is a group formed by 
+passionate batch system maintainers looking to support batch workload in cloud native environments.
+ 
+## Background
+
+More and more users need to migrate batch workloads to cloud native environments, and multiple existing 
+solutions from different community projects currently address this requirement, 
+e.g. Volcano, Yunikorn, Armada, MCAD, which makes it easy for users to migrate.
+But from a user perspective, the current support for batch workloads is widely different from 
+one batch system to another and requires a comprehensive migration between them. 
+It’s necessary to build a specification for batch workloads in the cloud native ecosystem, 
+and help related projects figure out their scopes and interaction. The users and communities can 
+follow this specification to work with projects in CNCF, e.g. Kubernetes, Volcano, Armada, MCAD, 
+or even out of CNCF, e.g. Yunikorn, and so on.
+  
+## Goal:
+
+The Cloud Native Batch System Initiative Working Group aims to define common batch system specifications in
+the cloud native ecosystem for different batch projects, and help related projects to figure out their scopes. 
```

Comment:
Should we be more clear about the output?
I think we can highlight that the output will be a group of Specs, CRDs, and client libraries.

Reply:
done :)